WebServer/error_handler.py

```diff
@@ -0,0 +1,9 @@
+from read_page_inf import read_page
+
+
+def error_page(self, s_path):
```

Comment:
the name for this function is too broad. It should either be named like `not_found_page` or whole method should be more generic: it should allow returning not only 404 error, but 400 too in case if form data is missing

Reply:
Corrected